package/clp.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import re
+
+def grab(url_name):
+  """
+  Scrapes the content from a cl1p.net clipboard and returns the text content.
+
+  Args:
+      url-name (str): the unique identifier for the clipboard URL.
+
+  Returns:
+      str: the text content from the clipboard or an error message.
+
+  Usage:
+      This function can be used as follows:
+      from package_name.clp import grab
+      grab('url_name')
+
+  Output:
+      Prints the contents of the page at cl1p.net/{url_name} if available.
+  """
+  
+  url = f"https://cl1p.net/{url_name}"
+  try:
+    with urllib.request.urlopen(url) as response:
+      html_content = response.read().decode('utf-8')
+    match = re.search(r'<textarea[^>]*id="cl1pTextArea"[^>]*>(.*?)</textarea>', html_content, re.DOTALL)
+    if match:
+      content = match.group(1)
+      content = re.sub(r'&[^;]+;', '', content)
+      content = re.sub(r'<[^>]+>', '', content)
+      content = re.sub(r'\s+', ' ', content).strip()
+      print(content)
+    else:
+      print("No content found on this clipboard.")
+  except urllib.error.URLError as e:
+    print(f"Failed to retrieve this page. Error: {e.reason}")
```

Comment:
so far everything is working just fine in the best case scenario(when I enter the existing page name)

I'll point the things I found!

### `else` condition:
- It only works when we use `grab("a")`
- Otherwise, it's just a blank output.

> for your reference
![image](https://github.com/user-attachments/assets/41e67143-b293-4dda-8fd1-69804a6a3cb6)
1. grab('a') -> exec else condition
2. grab('aaasdasdagafdgdf') -> exec nothing
3. grab('sneaky') -> working and displays the content.
> but still it only shows the single line text.